did_it_run/tests/lib_tests.rs

```diff
@@ -0,0 +1,35 @@
+extern crate did_it_run;
+
+const EXIT_WITH_ARGUMENT_COUNT: &str = "exit $#";
+const EXIT_WITH_STATUS: &str = "exit $0";
+
+#[test]
+fn exits_with_status() {
+    let base_args = vec!["bash", "-c", EXIT_WITH_STATUS];
+    for status_code in vec![-7, -1, 0, 1, 2, 254, 255, 256, 2000] {
```

Comment:
I think it's worth pulling out this list of numbers into a constant named status_codes. I think having this inline hurts readability.

Reply:
:+1: Good idea.